### ISSUE TITLE
Clicking Toast notification while MPC-HC is in focus opens file dialog

### ISSUE THREAD

**Issue by Zerowalker** [clsid2/mpc-hc#2447]
As topic says.

I noticed this odd bug, if i got the player in focus and there is some toast notification,
say a calendar event or something, and you click on it (not just close it), mpc-hc opens the "file dialog",
i believe it's the "Quick Open File".

It's not a big deal but thought it might be good to finally post the issue,
It could perhaps be something on my end, but not sure what that could be if that's the case,
but it should be fairly easy to test by doing the steps above.

**Comment by xLn2** [comment]
Might be related with `Global Media Keys` option that located at `Options > Player > Keys`.  You might try disable this setting, restart player and computer.

**Comment by adipose** [comment]
https://github.com/clsid2/mpc-hc/issues/2204

Sounds much like this.

**Comment by Zerowalker** [comment]
> Might be related with `Global Media Keys` option that located at `Options > Player > Keys`. You might try disable this setting, restart player and computer.

That one is already disabled.

**Comment by clsid2** [comment]
Options > Player > Keys
Change the keybinding of Quick Open File.

This is NOT a bug in MPC-HC.

You need to report it to Microsoft or to the developers of the app that shows the notification.

**Comment by adipose** [comment]
> As topic says.
> 
> 
> 
> I noticed this odd bug, if i got the player in focus and there is some toast notification,
> 
> say a calendar event or something, and you click on it (not just close it), mpc-hc opens the "file dialog",
> 
> i believe it's the "Quick Open File".
> 
> 
> 
> It's not a big deal but thought it might be good to finally post the issue,
> 
> It could perhaps be something on my end, but not sure what that could be if that's the case,
> 
> but it should be fairly easy to test by doing the steps above.

Does it happen every time? I had trouble reproducing the other bug since I had to wait for YouTube popups.

Windows version?

**Comment by Zerowalker** [comment]
Windows 10 22H2 19045.3803

I actually can't seem to reproduce it, tried triggering it with the "Clock" timer app.
So it's not always as I thought, will have to keep track next time it happens to see if it's a specific notification or something.

**Comment by adipose** [comment]
> Windows 10 22H2 19045.3803
> 
> I actually can't seem to reproduce it, tried triggering it with the "Clock" timer app. So it's not always as I thought, will have to keep track next time it happens to see if it's a specific notification or something.

Thanks. In the past it had to do with Youtube/chrome toast.  Something about clicking the notification was triggering a hotkey in mpc-hc, but it shouldn't be possible--unless it's just mass sending keystrokes to players registered with SMTC?

**Comment by Zerowalker** [comment]
It actually seems to be solved. Must have been solved quite recently.
Not sure if it's MPC-HC, Windows or something on my end.
But I'm quite sure the issue was there some months ago.
I have updated MPC-HC whenever there's been a new release, so I'm currently on 2.1.3

**Comment by clsid2** [comment]
If it comes back, let us know which app notification caused it.

**Comment by Zerowalker** [comment]
Okay found one that seems to trigger it.
When i got an google calender notification from chrome and click "snooze" it opens the file explorer.

**Comment by lastmetroid** [comment]
Oh wow, so I was right all the time and @clsid2 claimed I was imagining things claiming it was all related to media controls, where it was not. Experienced it again right now with Edge  twitter notifications, having an MPC instance open, clicking "dismiss" opens "save a copy" in MPC. I am on Windows 11 23h2 22631.3155 Edge 122 and MPC 2.1.6 and I can reproduce it with 100% getting a Edge twitter notification of a twitter DM. Clicking "dismiss" and having MPC in front opens "save a copy" dialog. Minimizing MPC doesnt trigger it. If another window is in front of mpc it also doesnt happen.

Had removed key binding for "quick open" and now ironically it opens "quick open" instead of "save a copy" when clicking dismiss of the Edge or Chrome notification.

![image](https://github.com/clsid2/mpc-hc/assets/6721083/da898e50-deeb-4743-ae99-323e548bbd3a)

![image](https://github.com/clsid2/mpc-hc/assets/6721083/9af69aa2-2ea0-436b-aedc-dd9f413c9caa)

**Comment by lastmetroid** [comment]
Ok this is definitely a bug in MPC. It seems if you clear a keybinding it doesnt clear it properly. 1st bug. And now comes the weird thing in combination:

I rebound quick open file with something else and now instead of quick open, clicking dismiss goes down to the next in the line and opens "open file/url" even it is empty.

![image](https://github.com/clsid2/mpc-hc/assets/6721083/b011a66a-6faa-4c7d-a42d-9bca117bf711)

![image](https://github.com/clsid2/mpc-hc/assets/6721083/dabdbbd6-398b-4d88-b3ce-31b835216a96)

If you now also change the 2nd line to something else like this:

![image](https://github.com/clsid2/mpc-hc/assets/6721083/80160781-462b-49ea-8a5f-fc0ca50e3d1c)

clicking dismiss on the toast widow with mpc on focus now opens the 3rd line open dvd/bd gui.

![image](https://github.com/clsid2/mpc-hc/assets/6721083/eaa997c4-17a2-4831-b61d-2b6aed273ae2)

If you now bind a key to dvd/bd, it goes to the next line opening device when pressing dismiss.

**Comment by clsid2** [comment]
I see ZERO evidence of a bug in MPC-HC.

Those functions can be triggered not just by a keybinding but also by sending a window message with their ID. So Edge must be sending messages to wrong window.

**Comment by lastmetroid** [comment]
@clsid2 please stop debating again where it is crystal clear this is a bug in MPC. The first one is maybe not a direct bug but I told you now the related BUG IN MPC.

![image](https://github.com/clsid2/mpc-hc/assets/6721083/622aff69-a187-449c-b57f-6c0131f01dce)

![image](https://github.com/clsid2/mpc-hc/assets/6721083/4c53e532-b0f3-43bf-bcf6-6922d4ce8654)

And this is a really dangerous bug.

1. bug inside mpc of not properly clearing key bindings
2.  key binding algorithm handling is bugged as seen above, jumping to next "empty" line and executing it

Chrome/Edge sending "something" to MPC, MPC executing the command from top to bottom first one which it finds with no binding in the key binding list.

if quick open file is not bound, it executes that. if it is bound and open file/url is not bound it executes that, if that one is bound and the next one not it executes that one, until it takes the one which is not bound.

**Comment by clsid2** [comment]
They should not send commands to MPC-HC window.

I can't reproduce with the notifications shown by Windows 11.

Here is test site for triggering notifications:
https://www.bennish.net/web-notifications.html

**Comment by lastmetroid** [comment]
That one is not working. You need a notification with buttons. Twitter for example then let someone send you a direct message with tab open of twitter page and without having dms open. the toast will have a message with a reply and dismiss button. clicking dismiss with mpc in focus triggers the issue.

You need a toast like this with a button under it

![image](https://github.com/clsid2/mpc-hc/assets/6721083/28299c08-bcaf-4314-a354-dc22874797f5)

**Comment by clsid2** [comment]
Ask Elon Musk to fix his crappy X. I am not using that.

**Comment by lastmetroid** [comment]
Who spit into your cornflakes when you were a child? Use something else Youtube and Edge and it will also have a dismiss button.

**Comment by lastmetroid** [comment]
This one works:

https://pushalert.co/demo

Use for example or others too
![image](https://github.com/clsid2/mpc-hc/assets/6721083/6ce67553-cd7a-4ebe-8a16-bf012a8be422)

clicking the mothers day specials button triggers the issue

**Comment by lastmetroid** [comment]
https://github.com/clsid2/mpc-hc/assets/6721083/b2ffe223-df15-4c0b-8672-7afb31800c6e

![image](https://github.com/clsid2/mpc-hc/assets/6721083/90a2fec3-dd17-4a79-962a-959371b43702)

It would execute line 1, it wont because bound, goes to 2nd line, it is bound, goes to 3rd line, goes to 4th line and exxecutes open device.

**Comment by clsid2** [comment]
That website does not trigger any actions in MPC-HC here. Clicking on notification just opens website as it should. I even tried setting MPC-HC to always on top.

**Comment by lastmetroid** [comment]
https://github.com/clsid2/mpc-hc/assets/6721083/6667410d-c90a-4d50-b542-8fc376324631

**Comment by lastmetroid** [comment]
Look what your "non existing bug" in MPC does if you have the key bindings bound like this:

![image](https://github.com/clsid2/mpc-hc/assets/6721083/058abf34-22bd-4fe4-bf3b-b2b59a9e328e)

It wants to delete the file if you click the toast button

https://github.com/clsid2/mpc-hc/assets/6721083/f645a828-f247-41ac-883e-ff43f7d043e5

**Comment by clsid2** [comment]
I think it might be sending some weird mouse action. Try this:
-removed-

**Comment by lastmetroid** [comment]
It says it contains a virus

![image](https://github.com/clsid2/mpc-hc/assets/6721083/9054459f-a054-41b7-9689-c724f9d0446d)

**Comment by clsid2** [comment]
Wacatac is common false positive by Defender.
https://www.virustotal.com/gui/file/e411daa8703569f74752e1eddd56a43b739a336151645210013e3e72bfd352e2

**Comment by lastmetroid** [comment]
No, the version doesnt fix the issue. tested in a virtualbox VM where the issue also happens.

https://pushpad.xyz/demo

![image](https://github.com/clsid2/mpc-hc/assets/6721083/f8dad808-1387-4b73-99e3-15c6777a2907)

**Comment by clsid2** [comment]
Try this one:
[mpc-hc64.zip](https://github.com/clsid2/mpc-hc/files/14467838/mpc-hc64.zip)
Will show a prompt if a command get triggered.

If that fails I have another idea to try.

**Comment by clsid2** [comment]
Might be fixed:
[mpc-hc64.zip](https://github.com/clsid2/mpc-hc/files/14467912/mpc-hc64.zip)

Seems like a bug in Windows.

**Comment by adipose** [comment]
Is it executing unbound actions?  Maybe sent globally with null action?

**Comment by adipose** [comment]
> please stop debating again where it is crystal clear this is a bug in MPC.

It may be a bug in mpc-hc or not.  It is certainly undesired behavior.  But as it was never an issue before Windows 10 toast popups, the question is whether it is rightly called a bug or simply designed without awareness of a future Windows feature, which may interfere with Accelerators.

https://learn.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-accel

If Windows is sending mpc-hc signals that are being interpreted as accelerators, is that a Windows bug or an mpc-hc bug?  And if it is a Windows bug, do we have the ability to identify when it was sent by windows or the user?  We must answer these questions to solve the problem and the dilemma!

**Comment by clsid2** [comment]
Are you able to reproduce it? Because I still can't.

I suspect that Windows is somehow matching against a null value in the accel list. That is of course a major bug. Plus the fact that is trying to send a (keypress) action to MPC window when pressing on a toast button is another bug.
But anyway, I have taken some measures against Windows stupidness that will hopefully fix the problem.

**Comment by lastmetroid** [comment]
> Might be fixed:

yes that last one seems to fix the issue

**Comment by adipose** [comment]
> Are you able to reproduce it? Because I still can't.

Nope, not so far.

**Comment by adipose** [comment]
With global media keys enabled, using the Chrome notifications, I can get a windows message `WM_HOTKEY` with `APPCOMMAND_MEDIA_PLAY`, even though the notification is only supposed to open a browser link.

**Comment by adipose** [comment]
Further, I was able to `APPCOMMAND_MEDIA_PAUSE` by disabling `APPCOMMAND_MEDIA_PLAY`.  It seems as if it sends a global APPCOMMAND to every window with the first registered APPCOMMAND.

**Comment by lastmetroid** [comment]
Like I said. This issue has nothing to do with media keys, never had media remote on in MPC as seen on the example videos above, it was obviously a bug in MPC which @clsid2 seems to has fixed. Not just that MPC executed commands by receiving something but also the list which goes through until finding an empty command binding and then executes that. maybe another bug in MPC that the option for global media keys is ignored.

**Comment by adipose** [comment]
> Like I said. This issue has nothing to do with media keys, never had media remote on in MPC as seen on the example videos above, it was obviously a bug in MPC which @clsid2 seems to has fixed. Not just that MPC executed commands by receiving something but also the list which goes through until finding an empty command binding and then executes that. maybe another bug in MPC that the option for global media keys is ignored.

I disagree.  There is no reason for windows to send global media keys appcommand to MPC-HC when you click on a chrome notification.  I just reproduced this.  Once I understand it better, I will submit a bug to Microsoft.

**Comment by lastmetroid** [comment]
> I disagree.

what do you disagree on again? so it s a magic trick that it happens with no option activated in MPC:

![image](https://github.com/clsid2/mpc-hc/assets/6721083/7b73408f-1403-44e7-bb27-586d3a7d2567)

**Comment by adipose** [comment]
@clsid2 ,

I have been able to find a kind of workaround for the issue I found.  If you use this (before registering other commands):

`RegisterHotKey(m_pMainWnd->m_hWnd, 0xFFFF, 0, 0);`

Then the chrome notifications send `0xFFFF` wParam instead.  This can be dropped the same as we drop wParam=0.

If you don't register it, the first one you register gets sent by the toast click.

**Comment by adipose** [comment]
> > I disagree.
> 
> what do you disagree on again? so it s a magic trick that it happens with no option activated in MPC:
> 
> ![image](https://private-user-images.githubusercontent.com/6721083/309487877-7b73408f-1403-44e7-bb27-586d3a7d2567.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkzOTgyMDIsIm5iZiI6MTcwOTM5NzkwMiwicGF0aCI6Ii82NzIxMDgzLzMwOTQ4Nzg3Ny03YjczNDA4Zi0xNDAzLTQ0ZTctYmIyNy01ODZkM2E3ZDI1NjcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDJUMTY0NTAyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGRkZGU0NWVkM2IxZTE2YmE1OTU0MzQ4YTcxZWExZTI0MjM0Zjk1NzlkN2Q5NTM5MGNmMDY2MmFhMWM1NjQyMyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.HQJxiNLEMCBgDm4FlhlKzbjVYYFd1EeQw1RF4r3qAUc)

The global media keys is one issue I can reproduce, but I cannot reproduce your specific issue.  If I could get access to your machine I'd happily debug what windows messages are coming to your app.

The fact that Windows sends garbage media keys when you click on a notification is enough to prove Microsoft has a major bug with how the toast notifications are being processed.  So I wouldn't be at all surprised if they did something else stupid, here.  Like, they see that you have NO registered app commands, so they instead send you other kinds of ACCEL commands in an attempt to "integrate" with running applications.

Bottom line, there is no reason to send me an app command because someone wants to click on a chrome popup.  But they do it anyway.

You are making assumptions without understanding how it works.  MPC-HC is REGISTERING hotkeys with Windows.  Windows, in turn, NOTIFIES MPC-HC that the key has been pressed.  We do not control Windows decisions on this.  When you clear a hotkey and the next hotkey runs instead, it may be a similar behavior to the one I just described about app commands (those are different, only work with global media keys, but similar concept).

**Comment by lastmetroid** [comment]
> If I could get access to your machine

what do you mean my machine? I can reproduce it with 100% chance on any Windows 11 PC, even in virtual machine in virtualbox as seen above.

**Comment by adipose** [comment]
I can never reproduce it.  Send me the VM that can repro it.

**Comment by lastmetroid** [comment]
I cant, it contains my Windows key and private information and the vm storage file is 60gb in size. It is a fresh Windows 11 installation though. I just installed it some days ago.

![image](https://github.com/clsid2/mpc-hc/assets/6721083/44c3a8f1-0813-4b85-b624-b1fa646a2bf7)

Send me a debug file I can execute in the vm which tells you the information you want.

**Comment by adipose** [comment]
In the past I couldn't reproduce this on windows 10 or 11.  Fresh doesn't mean you didn't install anything, etc.

Something is different for you but I don't know what, yet.

Clsid2 has a workaround but the actual bug is unclear.

**Comment by lastmetroid** [comment]
Nope. Nothing installed or changed on the VM

![image](https://github.com/clsid2/mpc-hc/assets/6721083/49d2e1b0-27e6-4e06-a609-2210e3ecbc98)

Thats all.

**Comment by adipose** [comment]
We can try team viewer or something.

**Comment by lastmetroid** [comment]
Tell me what I shall do to give you the information, what would Teamviewer be useful for.

**Comment by lastmetroid** [comment]
do you have telegram or discord chat (no voice call) or something maybe we can do tv but what would you want to do on it then

**Comment by adipose** [comment]
I would like to do remote debugging session so I can see what windows messages arrive at the time of bad behavior.

Logging is not as good but with some work I could create a build to record what happened.

**Comment by lastmetroid** [comment]
I asked how you would do that. do you have discord or telegram or not. give me yours so I can give you the teamiewer data.

**Comment by adipose** [comment]
> I asked how you would do that. do you have discord or telegram or not.

I have telegram.  But it is important to interact directly with the machine so the debugger can observe the process.

**Comment by makedir** [comment]
I told you now countless times to give me that so I can give you tv data.

**Comment by makedir** [comment]
I am waiting I have not infinite time.

**Comment by clsid2** [comment]
Maybe a tiny standalone demo app would be more useful for bug reporting to Microsoft. A basic MFC GUI app that registers some hotkeys and key accels. Then also easy to log all incoming commands.

**Comment by lastmetroid** [comment]
Tell me your telegram nick I have the virtual machine open with teamviewer.

**Comment by clsid2** [comment]
> I have been able to find a kind of workaround for the issue I found. If you use this (before registering other commands):
> 
> `RegisterHotKey(m_pMainWnd->m_hWnd, 0xFFFF, 0, 0);`
> 
> Then the chrome notifications send `0xFFFF` wParam instead. This can be dropped the same as we drop wParam=0.
> 
> If you don't register it, the first one you register gets sent by the toast click.

Good idea.
Still super weird that it uses first registered hotkey.

**Comment by lastmetroid** [comment]
> Still super weird that it uses first registered hotkey.

you mean first not registered.

![image](https://github.com/clsid2/mpc-hc/assets/6721083/983de18d-5b16-411b-b44e-bfd4dbc558fb)

it would execute "open dvd/bd" in this case

**Comment by clsid2** [comment]
I am talking about the other issue with unwanted hotkey command (APPCOMMAND_MEDIA_PLAY).

**Comment by makedir** [comment]
So is @adipose still here? I am waiting with Telegram and virtualbox open with teamviewer. I wanted to go to the supermarket I have maybe 30 mins time.

**Comment by adipose** [comment]
> Tell me your telegram nick I have the virtual machine open with teamviewer.

I don't post that publicly but if there is a DM function in github maybe we can do that.

**Comment by makedir** [comment]
NO THERE IS NOT

**Comment by makedir** [comment]
why do you think I am asking? change your telegram nick my god

**Comment by makedir** [comment]
HERE @mkxxyy123123123 telegram me my god you can change telegram nick endless times

**Comment by makedir** [comment]
I still didnt get any message in Telegram

**Comment by lastmetroid** [comment]
@clsid2 I showed the issue happening to @adipose on tv. maybe there is a confusion here because you couldnt reproduce it, you need to focus click one time again on MPC before clicking the final time on the toast button or it is not working. there was also a weird glitch not sure why, but it was not reproducible anymore until I rebooted Windows and reinstalled Chrome after @adipose clicked something in Chrome.

**Comment by clsid2** [comment]
Yes, I was already giving MPC focus when testing.

I think bug with global hotkeys is due to missing stuff in GetVKFromAppCommand() which previously caused 0 to get registered as hotkey. That is now fixed. I will add missing stuff to GetVKFromAppCommand() later today.
Adipose, can you test if also fixed for you with latest Git?

**Comment by adipose** [comment]
I have finally reproduced it with @makedir's help.  The issue is caused only when the following setting is disabled in Windows:

![image](https://github.com/clsid2/mpc-hc/assets/3324395/eaae444c-9f28-41c4-be68-2777b54c6c53)

It's unclear why this matters but the animation may cause the window focus to change enough to avoid the erroneous message.

**Comment by adipose** [comment]
> Yes, I was already giving MPC focus when testing.
> 
> I think bug with global hotkeys is due to missing stuff in GetVKFromAppCommand() which previously caused 0 to get registered as hotkey. That is now fixed. I will add missing stuff to GetVKFromAppCommand() later today. Adipose, can you test if also fixed for you with latest Git?

I could not reproduce either issue on the latest.

**Comment by clsid2** [comment]
Can reproduce it now.

When clicking the notification button, look at MPC-HC titlebar color. You see that it loses focus, gains it, and loses it again. So it is clear that window messages are being send. Without the animate controls tweak disabled, it does not regain focus.

No unwanted actions anymore though after my adjustments in latest dev build. So problem solved. I can probably turn on global hotkeys by default again as well now.

**Comment by adipose** [comment]
Yes, I think so.  Here's the behavior as I can see it:

1. ...
2. The window with focus is sent a WM_KEYFIRST message with wParam=0.  This is a side effect of having registered an accelerator with key=0.  In this case it will match the first "unbound" key in the table as they all have a key=0.

Neither of these behaviors makes any sense to me.  But perhaps the media controls capabilities of Windows always sends two commands, one APPCOMMAND and one virtual key, but when there is no media control to take, it just sends zero for both???

Crazy.

**Comment by adipose** [comment]
I understand the APPCOMMAND issue better now.  It was getting `APPCOMMAND_MEDIA_PLAY` which has no virtual key, so it was registering for key zero as well.

So basically, every time you hit a windows notification, it must press key = 0, which triggers both global keys and the window in focus in different ways.  And we unfortunately matched on both.

**Comment by lastmetroid** [comment]
> And we unfortunately matched on both.

So it was all a bug (multiple) in MPC at all.

**Comment by clsid2** [comment]
Yes and no. Because it only triggers bug situation due to the bug in Windows.

**Comment by adipose** [comment]
We can call it a bug that keystroke zero is generated at all.  We can also say that zero is not a valid binding per [here](https://learn.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes).  But if it is not a valid binding, Microsoft should also not generate such keystrokes...

**Comment by adipose** [comment]
The bug is caused by chrome.

In `chromium\chrome\notification_helper\notification_activator.cc`, the following code (with comments to explain why) occurs:

```
    // Despite the fact that the Windows notification center grants the helper
    // permission to set the foreground window, the helper fails to pass the
    // baton to Chrome at an alarming rate; see https://crbug.com/837796.
    // Sending generic down/up key events seems to fix it.
    INPUT keyboard_inputs[2] = {};

    keyboard_inputs[0].type = INPUT_KEYBOARD;
    keyboard_inputs[0].ki.dwFlags = 0;  // Key press.

    keyboard_inputs[1] = keyboard_inputs[0];
    keyboard_inputs[1].ki.dwFlags |= KEYEVENTF_KEYUP;  // key release.

    ::SendInput(2, keyboard_inputs, sizeof(keyboard_inputs[0]));

```

I was able to reproduce the issue with a generic toast and calling this code after click.  Hidden in the INPUT type is a union that represents a `KEYBDINPUT` in the case of `INPUT_KEYBOARD`.  As you might expect, it has a member called `wVk` which is not being initialized here (thus will be zero on most compilers), and which represents a virtual key code.  Hence our mystery keystroke comes from a Chrome workaround for what is apparently a windows bug.  They did not deliberately select vk=0, but given what they are doing, I'm not sure any other code would be preferred.  Garbage in, garbage out, from our end, though.

Thanks to @makedir for the suggestion to check if the bug originates in Chrome.

Furthermore, the disabling of the animation may well be the thing that triggers Chrome's bug, which means that Windows has a bug handing off focus from toast notifications, when the animation is disabled.

**Comment by lastmetroid** [comment]
Anyone also gets 99% cpu usage opening this bug report github link? 

![image](https://github.com/clsid2/mpc-hc/assets/6721083/fb20e4b4-609a-4715-80fb-867915848eff)

**Comment by xLn2** [comment]
> Anyone also gets 99% cpu usage opening this bug report github link?

It may be related to Chrome based browsers or github. I had a lot of browser crashes when I was preparing Issues, but fortunately I haven't encountered them lately.

**Comment by clsid2** [comment]
Time to move to Firefox ;)
